ingest/importer/submission/entity.py

```diff
@@ -1,14 +1,15 @@
 # TODO Check how we can refactor to merge MetadataEntity and Entity
 class Entity(object):
     def __init__(self, entity_type, entity_id, content, ingest_json=None, links_by_entity=None,
-                 direct_links=None, is_reference=False, is_linking_reference=False, linking_details=None, concrete_type=None,
+                 direct_links=None, external_links=None, is_reference=False, is_linking_reference=False, linking_details=None, concrete_type=None,
```

Comment:
To me this `Entity` object's constructor has way too many parameters. It would be better to refactor it considering object composition.